README.md

```diff
@@ -1,3 +1,39 @@
 ## galaxy-integrations-updater
 
 Autosync FriendsOfGalaxy forks with corresponding base repo integration for GOG Galaxy 2.0.
+
+
+### Setup requirements for integration
+
+- is deployed on Github
+- has release branch named `fog_release` (other rules apply only to this branch; if it doesn't exists default branch may be used)
+- dependencies
+    - those not installable by pip should be commited to the git repository in ready-to-use state
+    - those installable by pip are stored in standard pip format in `requirements/app.txt` in the repository root (`requirements.txt` may also used)
+        - dependencies are pinned (only using '==') - this is for easier security checks and building final package by us
+- has valid `manifest.json` with all fields listed in Galaxy Plugin Python API documentation
+    - "version" - in Semantic Versioning but without extensions like "beta" - to be comparable
+    - "script" - cannot link to upper directory file
+
+### Build process
+
+`fog_release` branch is searched for `manifest.json` location automatically. The direct parent of `manifest.json` is treated as *source directory* and only this is copied to the final build during release package preparation.
```

Comment:
> and only this is copied to the final build during release package preparation.
I think this may be confusing,  something more like : 

> The direct parent of `manifest.json` is treated as *source directory*. Content of source directory along with all of its subdirectories is copied to final build during release package preparation.